src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Symbols/MethodOrPropertySymbol.cs

```diff
@@ -171,7 +171,10 @@ public bool MarshalAsObject(int index)
             if (IsMarshalAsParameter(index))
             {
                 UnmanagedType marshalAsType = GetMarshalAsParameterValue(index);
-                return marshalAsType == UnmanagedType.Interface || marshalAsType == UnmanagedType.IUnknown || marshalAsType == UnmanagedType.IDispatch;
+                return marshalAsType == UnmanagedType.Interface || marshalAsType == UnmanagedType.IUnknown ||
+#pragma warning disable CS0618 // obsolete
```

Comment:
@terrajobst, @AaronRobinsonMSFT, this surprised me.  UnmanagedType.IDispatch is not obsolete in netstandard2.0 nor netcoreapp3.0, but is in netstandard2.1.  Was that an explicit choice?